ci/long_running_tests/start_workloads.sh

```diff
@@ -70,17 +70,6 @@ popd
 echo ""
 echo ""
 
-echo "To kill the instances, use the following commands."
-echo ""
-for workload_file in "$ROOT_DIR"/workloads/*; do
-  file_name=$(basename -- "$workload_file")
-  workload_name="${file_name%.*}"
-  echo "    ray down -y $ROOT_DIR/$CLUSTER_CONFIG --cluster-name=$workload_name"
-done
```

Comment:
You don't find this message helpful?

Reply:
It just led to accidentally deleting the nodes when trying to attach to it...

Reply:
Ok